src/domain/occurrence/enrolmentDetails/EnrolmentDetails.tsx

```diff
@@ -153,25 +153,25 @@ const EnrolmentDetails: React.FC<EnrolmentDetailsProps> = ({
     }
   };
 
-  const getStudyLevel = () => {
-    const studyLevel = enrolment?.studyGroup?.studyLevel;
+  const getStudyLevels = (studyLevels: StudyLevelNodeConnection): string => {
+    if (studyLevels) {
+      const translations: string[] | undefined = studyLevels.edges?.map(
+        (edge) => {
+          if (!edge?.node?.id) return '';
+          const level: string = edge.node.id.toUpperCase();
 
-    const getStudyLevelString = (studyLevel: StudyLevel) => {
-      return studyLevel.startsWith('GRADE')
-        ? t('studyLevel.grade_interval', {
-            postProcess: 'interval',
-            count: Number(studyLevel.split('_')[1]),
-          })
-        : translateValue('studyLevel.', studyLevel, t);
-    };
-
-    if (Array.isArray(studyLevel)) {
-      return studyLevel.map((level) => getStudyLevelString(level)).join(', ');
+          return level.startsWith('GRADE')
+            ? t('studyLevel.grade_interval', {
+                postProcess: 'interval',
+                count: Number(level.split('_')[1]),
+              })
+            : translateValue('studyLevel.', level, t);
+        }
+      );
+      return translations.join(', ');
```

Comment:
Common parts with studyLevelOptions in EnrolmentForm.tsx could be maybe extracted to some common helper function 🤔  (yes I noticed that the duplications were there even before your changes :))

Reply:
A new studyLevel util is now created in commit https://github.com/City-of-Helsinki/palvelutarjotin-admin/commit/ef2bbc9798e6463c7105309798b03d8b5bc8c8f3. Commit is included in pull request !139, which is also currently rebased to this PR's branch, **which makes this PR obsolete!**